speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp

```diff
@@ -76,6 +76,8 @@ const char strFrontWing[] = "Front wing";
 const char strRearWing[] = "Rear wing";
 const char strPenalty[] = "Next pit type";
 
+ssgSimpleState* interventionTexture;
```

Comment:
This is a change to SD. It is recommended to add a comment above every SD change.
use this style (cause I used this)
// SIMULATED DRIVING ASSISTANCE CHANGE: what is different